src/CommandAndStateMessageParser.cpp

```diff
@@ -597,8 +597,11 @@ samples::Joints CommandAndStateMessageParser::getCameraHeadTiltMotorState(string
 
     samples::Joints motor_states;
     motor_states.time = Time::now();
+    auto tilt = root["tilt"]["position"].asDouble() * M_PI / 180;
```

Comment:
camera_head2world_tilt

And use base::Angle in the calculations to make sure the end position is normalized (given that the head can move almost 180, that's definitely needed)